src/cryptography/hazmat/bindings/openssl/binding.py

```diff
@@ -4,10 +4,69 @@
 
 from __future__ import absolute_import, division, print_function
 
+import os
 import threading
 
 from cryptography.hazmat.bindings._openssl import ffi, lib
 
+_osrandom_engine_id = ffi.new("const char[]", b"osrandom")
+_osrandom_engine_name = ffi.new("const char[]", b"osrandom_engine")
+
+@ffi.callback("int (*)(unsigned char *, int)", error=0)
+def osrandom_rand_bytes(buf, size):
+    signed = ffi.cast("char*", buf)
+    result = os.urandom(size)
+    signed[0:size] = result
+    return 1
+
+
+@ffi.callback("int (*)(unsigned char *, int)", error=0)
+def osrandom_pseudo_rand_bytes(buf, size):
+    result = osrandom_rand_bytes(buf, size)
+    if result == 0:
+        return -1
```

Comment:
So, this is (poorly) mimicking the behavior of the C code, which I'm pretty sure was also never exercised.  I don't understand why the FIPS thing does what it does, and I'm inclined to just make `osrandom_rand_bytes` be `method.pseudorand` as well and just delete this function.  Can someone - @reaperhulk maybe - tell me why that would be a bad idea?